ddtrace/contrib/aiohttp/template.py

```diff
@@ -0,0 +1,28 @@
+"""
+Instrumenting aiohttp_jinja2 external module
+TODO: better docstring
+"""
+import aiohttp_jinja2
+
+from ddtrace import Pin
+
+
+def _trace_template_rendering(func, module, args, kwargs):
+    """
+    Trace the template rendering
+    """
+    # get the module pin
+    pin = Pin.get_from(aiohttp_jinja2)
+    if not pin or not pin.enabled():
+        return func(*args, **kwargs)
+
```

Comment:
will introduce a signature protection here; in case the ``render_template()`` will be changed in future `aiohttp_jinja2` releases